tests/integration_tests/tests/agentless_tests/test_depends_on_lifecycle_operation.py

```diff
@@ -80,14 +80,14 @@ def _test_full_flow(self, expected_info, tested_operation):
                 operations_id[op.id]['dependencies'] = op.dependencies
                 operations_id[op.id]['info'] = op.info
 
-                task_kwargs = op.parameters.get('task_kwargs', {})
-                if task_kwargs.get('cloudify_context', None):
-                    node_task = op.parameters['task_kwargs']
-                    cloudify_context = node_task['cloudify_context']
-                    op_name = cloudify_context['operation']['name']
-                    node_name = cloudify_context['node_name']
-                    operations_info[(op_name,
-                                     node_name)] = op.containing_subgraph
+                try:
+                    cloudify_context = op.parameters['task_kwargs'][
```

Comment:
What do u say about doing ".get"?

Reply:
it would have to be a chain of 3 get calls, so this is better